gf-ui-lib/src/components/Button/Button.tsx

```diff
@@ -16,19 +17,29 @@ interface ButtonProps {
    * Button type
    */
   type: "submit" | "button";
```

Comment:
Could u make type prop optional too, please? I suggest to button as default value

Reply:
Done